R/helper.R

```diff
@@ -0,0 +1,24 @@
+# Expansion of base::difftime for months
+time.frequency = function(time) {
+  diff = diff(time)
+  diff.unique = unique(diff)
+  attr(diff.unique, "units") = attr(diff,"units")
+  if(length(diff.unique) == 1){
+    return(diff.unique)
+  }
+  time = as.POSIXlt(time)
+  months = time$year*12 + time$mon
+  diff.months = unique(diff(months))
+  if(length(diff.months) == 1){
+    attr(diff.months, "units") = "months"
+    return(diff.months)
+  }
+  return(diff.unique)
+}
```

Comment:
Is this for a class called `frequency` or is that just using dot in the name? I'd recommend using underscores instead of periods as periods after functions or objects are also used for S3. It may also be worth having a method for nanotime and business days